packages/shared/src/components/modals/generic/common.ts

```diff
@@ -18,19 +18,4 @@ export const promotion: Record<string, MarketingCta> = {
       tagText: 'New Release',
     },
   },
-  bookmarkPromoteMobile: {
```

Comment:
Can we also remove modal component or do we use it somewhere else?

Reply:
It's the generic marketing CTA one :)